src/lib/npm.ts

```diff
@@ -108,125 +113,119 @@ export const getLatestVersionOfNPMDependency = async (
-      `Not able to install ${name}@${version} to ${dependencyDirPath}. \n${stringifyUnknown(
-        error,
-      )}`,
-    );
+
+    return;
   }
-};
 
-type EnsureNpmDependencyArg = {
-  nameAndVersion: string;
-  maybeFluenceConfig: FluenceConfig | null;
-  global?: boolean;
-  force?: boolean | undefined;
-  explicitInstallation?: boolean;
-};
+  const packageNameAndVersionTuple = splitPackageNameAndVersion(
+    packageNameAndVersion,
+  );
 
-export const ensureNpmDependency = async ({
-  nameAndVersion,
-  maybeFluenceConfig,
-  global = true,
-  force = false,
-  explicitInstallation = false,
-}: EnsureNpmDependencyArg): Promise<string> => {
-  const [name, maybeVersion] = splitPackageNameAndVersion(nameAndVersion);
-
-  const resolveVersionToInstallResult = await resolveVersionToInstall({
-    name,
-    maybeVersion,
-    packageManager: "npm",
-    maybeFluenceConfig,
-  });
+  const [packageName] = packageNameAndVersionTuple;
+  let [, version] = packageNameAndVersionTuple;
 
-  const version =
-    "versionToInstall" in resolveVersionToInstallResult
-      ? resolveVersionToInstallResult.versionToInstall
-      : await getLatestVersionOfNPMDependency(
-          resolveVersionToInstallResult.maybeVersionToCheck === undefined
-            ? name
-            : `${name}@${resolveVersionToInstallResult.maybeVersionToCheck}`,
-        );
-
-  const { dependencyDirPath, dependencyTmpDirPath } =
-    await resolveDependencyDirPathAndTmpPath({
-      name,
-      packageManager: "npm",
-      version,
-    });
+  if (version === undefined) {
+    version = await getLatestVersionOfNPMDependency(packageName);
+  }
 
-  await handleInstallation({
-    force,
-    dependencyDirPath,
-    dependencyTmpDirPath,
-    explicitInstallation,
-    name,
-    version,
-    installDependency: () => {
-      return installNpmDependency({
-        dependencyDirPath,
-        dependencyTmpDirPath,
-        name,
-        version,
-      });
+  await runNpm({
+    args: ["i", packageNameAndVersion],
+    options: {
+      cwd: await ensureFluenceAquaDependenciesPath(),
     },
   });
 
-  await updateConfigsIfVersionChanged({
-    maybeFluenceConfig,
-    name,
-    version,
-    global,
-    packageManager: "npm",
-  });
+  fluenceConfig.dependencies.npm[packageName] = version;
+  await fluenceConfig.$commit();
+}
 
-  await addCountlyLog(`Using ${name}@${version} npm dependency`);
-  return join(dependencyDirPath, NODE_MODULES_DIR_NAME);
+type NpmUninstallArgs = {
+  packageNameAndVersion: string;
+  fluenceConfig: FluenceConfig;
 };
 
-type InstallAllNPMDependenciesArg = {
-  maybeFluenceConfig: FluenceConfig | null;
-  force?: boolean | undefined;
-};
+export async function npmUninstall({
+  packageNameAndVersion,
+  fluenceConfig,
+}: NpmUninstallArgs) {
+  assertDependenciesPresent(fluenceConfig);
+  const [packageName] = splitPackageNameAndVersion(packageNameAndVersion);
+
+  await runNpm({
+    args: ["un", packageNameAndVersion],
```

Comment:
```suggestion
    args: ["uninstall", packageNameAndVersion],
```
Looks a bit clearer.